kentikapi/internal/testutil/http.go

```diff
@@ -88,6 +89,7 @@ func (h *MultipleResponseSpyHTTPHandler) ServeHTTP(rw http.ResponseWriter, r *ht
 
 	rw.Header().Set("Content-Type", "application/json")
 	response := h.response()
+	time.Sleep(h.handlingDelay)
```

Comment:
Nit: I'd prefer that line above starting forming the response (i.e. above `rw.Header().Set("Content-Type", "application/json")`), not in the middle of it, to increase the cohesion of the block of code. Possible with a line break between them.

Reply:
Done